pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -440,6 +443,20 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1beta1.PipelineRun) err
 		})
 		return nil
 	}
+	// Ensure that the PipelineRun provides all the parameters required by the Pipeline
+	err = resources.ValidateRequiredParametersProvided(&pipelineSpec.Params, &pr.Spec.Params)
+	if err != nil {
+		// This Run has failed, so we need to mark it as failed and stop reconciling it
+		pr.Status.SetCondition(&apis.Condition{
```

Comment:
please use `pr.Status.MarkFailed(` instead of `pr.Status.SetCondition` which @mattmoor added in PR #2741, something similar to: 
https://github.com/tektoncd/pipeline/blob/master/pkg/reconciler/pipelinerun/pipelinerun.go#L382